### DISCUSSION TITLE
KB5036980 update caused a problem

### DISCUSSION THREAD

**Discussion by kwagraphics** [valinet/ExplorerPatcher#3166] [Q&A]
I'm not sure if this is the right place for this post, but here goes!

I use Explorer Patcher v22621.3296.641 and StartAllBack 3.7.8. I figured out how to have a Windows 10 style start menu using them together. Been very happy with the look and usability of Windows 11 with the 3rd party software solutions I've collected over the years.

I had to reinstall the Windows 11 in early March for 2 laptops. They had different corrupt system32 files and it had to be done. I have Win 11 pro 23h2 22631.3447 and all drivers and software are up-to-date. 

April 23rd I received the kb5036580 update and it ruined everything! I've been searching different sites to see if anyone else has had a problem with the update and if there is a solution, but no joy!

My start menu changed to Windows 11 after the update and I couldn't find anyway to get it back the way it was. I uninstalled StartAllBack and tried Start11 to see if that would work, but it didn't change the start menu either. Using Start11 previously I was able to make changes to the Windows 11 start menu that made it useable (still don't like it), but that didn't work after the update either. No way to change the new start menu. So, back to StartAllBack!

StartAllBack's Windows 7 style start menu's still work! I don't like the Windows 7 start menu's after using the Windows 10 style start menu for so many years. I thought I'd give it a try though. I set it up, but I just couldn't stand it.

I had to un-install the update and everything is back to normal. I know Windows update will keep trying to give me this update. I've turned off updates for now and I don't have a problem getting my updates from the Windows catalog, but I don't want this to be a permanent solution.

This problem may be unique to me. Am I using EP and StartAllBack in a unique way that no one else is doing? The kb5036580 update is a problem. If no one else is using EP and SAB the way I do, I may be alone with no solution.

FYI: the way get the Windows 10 start menu and all the SAB benefits is to simply turn off "Use enhanced classic start menu" in SAB. Use EP to select the Windows 10 start menu. And Bob's your uncle! Until now....

**Comment by Scabad** [comment]
I too have experienced the same problem after updating Windows with KB5036980 the start menu will not allow me to use the Windows 10 style even though ExplorerPatch says that it is set to Windows 10, along with @kwagraphics if anyone has any suggestions I'd really appreciate any help, in resolving this problem.

Thanks in advance
Scabad

**Reply by kwagraphics** [reply]
I've been doing some more research into this and Windows 11 is trying (and succeeding) in disabling 3rd party software like Explorer Patcher and Start All Back. The next big Windows 11 update will be a huge problem. I learned there is a work-around installing 3rd party customization software. Rename the .exe.

I just don't know why MS is doing this. The version of the OS interface they are forcing on me interferes in my usability. I have it set up to function the way I want and need.

**Reply by pan-ch** [reply]
Same problem here. I was not successful with renaming the .exe (program and service in registry). 
The most awful start menu is still here.

I think MS wants to really create a "Windows-Vista-11"...

**Comment by KaotikPaladin** [comment]
Same problem here.

**Comment by pbnotts** [comment]
KB5036980 has knocked out the left, right and up arrows from File Manager. I have not come across any other issue so far. I uninstalled KB5036980 and the arrows re-appeared. I also run StartisBack but did not check whether this was affected.

**Reply by pbnotts** [reply]
Thanks for the response.  I've now unticked "Get the latest updates etc". I've never had any problem with them before.

**Comment by kp-gl** [comment]
KB5036980  is a optional/preview update. Don't install preview updates.
See https://github.com/valinet/ExplorerPatcher/wiki/Symbols

**Reply by pbnotts** [reply]
Thanks for the response.  I've now unticked "Get the latest updates etc". I've never had any problem with them before.

**Comment by pyrates999** [comment]
In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

Preview updates are also not fully tested by Microsoft and thus could have bugs.

You should only be installing preview updates if you are a developer or are OK with running pre-release versions of EP.
